base/sysinfo.jl

```diff
@@ -117,7 +117,7 @@ function _cpu_summary(io::IO, cpu::Array{CPUinfo}, i, j)
     println(io)
 end
 
-function cpu_summary(io::IO=STDOUT, cpu::Array{CPUinfo}=cpu_info())
+function cpu_summary(io::IO=STDOUT, cpu::Vector{CPUinfo}=cpu_info())
```

Comment:
`Array` seems like a wrong argument specialization; I would think that perhaps either `AbstractArray{CPUinfo}`, `AbstractVector{CPUinfo}`, `Vector{CPUinfo}`, or `Any` is better.